kairon/shared/llm/gpt3.py

```diff
@@ -40,20 +40,22 @@ def __init__(self, bot: Text, llm_settings: dict):
         self.__logs = []
 
     def train(self, *args, **kwargs) -> Dict:
-        self.__create_collection__(self.bot + self.suffix)
         self.__create_collection__(self.bot + self.cached_resp_suffix)
         count = 0
         contents = list(CognitionData.objects(bot=self.bot))
```

Comment:
write a group by collection query instead of this. Then for each collection name, create a collection and iterate over its contents to get embeddings and push those to the collection